package.json

```diff
@@ -7,12 +7,25 @@
   "author": "Akshay S Dinesh <asdofindia@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "jspm bundle src/ascii-unicode-converter.js dist/bundle.js --skip-source-maps"
+    "run-server": "node_modules/http-server/bin/http-server --cors -p 9000",
+    "postinstall": "cp node_modules/siimple/dist/*.min.css src/assets/ && cp node_modules/siimple-colors/dist/*.min.css src/assets/",
+    "build:css": "cp -R src/assets/ dist/",
```

Comment:
Why I'm feeling queasy about it is,

1. I'm used to seeing src folder have contents that is actually authored by the authors of the project.
2. The way this is now, right after `git clone && npm install` a fresh copy of the repository has untracked files.

There are two ways of adding third party code to repos that I've seen and grown comfortable with:
1) Having a `vendor` folder (which could be inside `src` or in root) consisting of third party code, and it's committed into git. This approach is usually used by projects where such dependencies are manually maintained and not frequently updated.
2) Not committing third party code into git at all, rather, using package.json or other dependency manager to keep track of the code. These are then referenced from build scripts and they become part of the generated code.